sa4u_z3/main.py

```diff
@@ -388,6 +388,60 @@ def TERM_signal_handler(sig_num: int, _frame):
     sys.exit()
 
 
+def handle_serialized_TU(tu: SerializedTU):
+    global solver
+    tmp_solver = Solver()
+    tmp_solver.from_string(tu.solver)
+    solver.add(tmp_solver.assertions())
+    return [Const(s, BoolSort())
+            for s in tu.assertions]
+
+
+def child_walkers(input: multiprocessing.Queue, output: multiprocessing.Queue, compilation_database_path: str, analysis_dir: Optional[str] = None) -> None:
+    global tu_assertions, tu_solver
+    initialize_z3()
+    tu_solver = solver
+    cindexDict = cindexDictMaker(compilation_database_path)
+
+    for filePath in iter(input.get, None):
+        compile_command = cindexDict[filePath]
+        try:
+            tu = parse_tu(compile_command, analysis_dir)
+        except:
+            print()
+        
+        ignore_locations = get_ignore_lines(tu)
+        cursor: cindex.Cursor = tu.cursor
+        tu_solver = Solver()
+        tu_assertions = []
+        walk_ast(
+            cursor,
+            walker,
+            {
+                'Seen': set([]),
+                'IgnoreLocations': ignore_locations,
+            },
+        )
+
+        stu = serialize_tu(tu, tu_solver, tu_assertions)
+
+        if analysis_dir:
+            write_tu(analysis_dir, stu)
+        
+        output.put(stu)
+    output.put(None)
+
+
+def cindexDictMaker(compilation_database_path: str,) -> dict[str, cindex.CompileCommand]:
```

Comment:
Methods should be named with `snake_case`, and let's use `Dict` instead of `dict` for consistency with the rest of the file.

Reply:
Done